Tests/Tests UI/test_base_page.py

```diff
@@ -0,0 +1,44 @@
+from Tests.PageObject.BasePage import BasePage
+from Tests.PageObject.locators import BasePageLocators
+
+base_link = "http://parabank.parasoft.com"
+
+
+def test_guest_use_buttons_gohome_contact_aboutus_from_register_page(browser):
+    page = BasePage(browser, base_link)
+    page.open()
+    page.button_go_home_is_working()
+    page.open()
+    page.click_on_button(*BasePageLocators.BUTTON_REGISTER)
+    page.button_go_contact_is_working()
+    page.open()
+    page.click_on_button(*BasePageLocators.BUTTON_REGISTER)
+    page.button_go_aboutus_is_working()
+
+
+class TestUpMenu:
+    def test_guest_use_button_about_us(self, browser):
```

Comment:
please add a fixture that would run before all tests and start the browser and execute commands

page = BasePage(browser, base_link)
        page.open()
        
So the browser would be loaded only once before all tests